Makefile

```diff
@@ -15,3 +15,20 @@ install.tools: .install.gitvalidation
 
 .install.gitvalidation:
 	go get -u github.com/vbatts/git-validation
+
+# Generate json-schemas.
+.PHONY: schemas
+schemas:
+	find schema -path 'schema/test-fixtures' -prune -o -name generate.jsonnet -execdir jsonnet -J jsonnet -m . {} \;
+
+.PHONY: embed-schemas
+embed-schemas:
+	printf "%s\n\n%s\n" "$$(cat .header)" "$$(go generate schema/gen.go)" > schema/fs.go
+
+.PHONY: test-fixtures
+test-fixtures:
+	find schema -path 'schema/test-fixtures/**' -name generate.jsonnet -execdir jsonnet -J jsonnet -m . {} \;
```

Comment:
Why use find here?

Reply:
This was a copypasta from my other project where I had a bunch of `generate.jsonnet` files in the schema directory. I can fix that.